packages/clerk-js/src/ui/components/UserProfile/__tests__/AccountPage.test.tsx

```diff
@@ -110,5 +110,111 @@ describe('AccountPage', () => {
       screen.getByText(/Enterprise Accounts/i);
       screen.getByText(/Okta Workforce/i);
     });
+
+    it('shows only the enterprise accounts of the user', async () => {
+      const emailAddress = 'george@jungle.com';
+      const firstName = 'George';
+      const lastName = 'Clerk';
+
+      const { wrapper } = await createFixtures(f => {
+        f.withEmailAddress();
+        f.withPhoneNumber();
+        f.withSocialProvider({ provider: 'google' });
+        f.withSaml();
+        f.withUser({
+          email_addresses: [emailAddress],
+          saml_accounts: [
+            {
+              id: 'samlacc_foo',
+              provider: 'saml_okta',
+              email_address: emailAddress,
+              first_name: firstName,
+              last_name: lastName,
+              saml_connection: {
+                id: 'samlc_foo',
+                active: true,
+                disable_additional_identifications: true,
```

Comment:
@NicolasLopes7 What do you think of wrapping those two tests into a `describe` block mentioning the `disable_additional_identifications` config? It helps to visualize that those two are in the same context. 

Something like: 
```
describe("with `disable_additional_identifications`", () => { 
  .....
})